src/Sarif.FunctionalTests/SpecExamples/RuleMetadata.sarif

```diff
@@ -3,15 +3,25 @@
   "runs": [
     {
       "tool": {
-        "name": "CodeScanner",
+        "name": "BinaryAnalyzer",
         "semanticVersion": "2.1.0"
       },
       "rules": {
-        "C2001": {
-          "id": "C2001",
-          "fullDescription": "A variable was used without being initialized. This can result in runtime errors such as null reference exceptions"
+        "BA2006": {
+          "id": "BA2006",
+          "name": "BuildWithSecureTools",
+          "shortDescription": "Application code should be compiled with the most up-to-date tool sets ...",
```

Comment:
Because without them, `fullDescription` is the same as `shortDescription`. I just wanted to indicate that `fullDescription` would really be longer. I'll remove the ellipsis from `shortDescription` and from the `messageFormats` below. I'll ponder what to do about `fullDescription`.

---
In reply to: [72288078](https://github.com/Microsoft/sarif-sdk/pull/464#discussion_r72288078) [](ancestors = 72288078)